.pre-commit-config.yaml

```diff
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.9.2
+    hooks:
+      - id: flake8
+        args: [--config=setup.cfg]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: no-commit-to-branch
+      - id: check-added-large-files
```

Comment:
[By default](https://github.com/pre-commit/pre-commit-hooks#check-added-large-files) this forbids file above 500kB to be added to the repo. In case we need to at some point, we can increase this limit.

Reply:
Was that ever a problem? No strong opinion

Reply:
Nope, but I don't want it to become one. Still, also no strong opinion. I'll remove it for now.